db/migrate/20221227062210_add_column_to_post.rb

```diff
@@ -0,0 +1,7 @@
+class AddColumnToPost < ActiveRecord::Migration[6.1]
+  def change
+    add_column :posts, :oc, :boolean
+    add_column :posts, :spoiler, :boolean
+    add_column :posts, :nsfw, :boolean
+  end
+end
```

Comment:
you seem to have 3 migration files just for this new feature. Is it really required?
Can you use just one file instead?

Reply:
I was not clear at first about using the columns for the features. So, I created many files. I ensure, next time I will analyse and create a single file and use effectiently.

Reply:
Can you delete the unwanted files now?